src/core/coap/coap_message.hpp

```diff
@@ -820,6 +904,16 @@ class Message : public ot::Message
         kBlockNumMax = 0xffff,
     };
 
+#if OPENTHREAD_CONFIG_COAP_BLOCKWISE_TRANSFER_ENABLE
+    OT_TOOL_PACKED_BEGIN
+    struct BlockWiseData
+    {
+        uint32_t       mBlockNumber;
+        bool           mMoreBlocks;
+        otCoapBlockSzx mBlockSize;
+    } OT_TOOL_PACKED_END;
```

Comment:
Do we need this to be packed? It seems to be used in `HelpData` only and we don't care about `HelpData` not being packed (not using packed when possible would help reduce code-size).